plugins/woocommerce/templates/myaccount/orders.php

```diff
@@ -68,7 +68,8 @@
 
 								if ( ! empty( $actions ) ) {
 									foreach ( $actions as $key => $action ) { // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
-										echo '<a href="' . esc_url( $action['url'] ) . '" class="woocommerce-button' . esc_attr( $wp_button_class ) . ' button ' . sanitize_html_class( $key ) . '">' . esc_html( $action['name'] ) . '</a>';
+                                        /* translators: 1: button label 2: order number */
+                                        echo '<a href="' . esc_url( $action['url'] ) . '" class="woocommerce-button' . esc_attr( $wp_button_class ) . ' button ' . sanitize_html_class( $key ) . '" aria-label="' . esc_attr( sprintf( __( 'View order number %s', 'woocommerce' ), $order->get_order_number() ) ) . '">' . esc_html( $action['name'] ) . '</a>';
```

Comment:
Getting another PHPCS error related to using spaces instead of tabs for indentation: https://github.com/woocommerce/woocommerce/actions/runs/9908739007/job/27375395683?pr=49424#step:4:27

Reply:
@dkotter this should now be fixed